coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1,56 +1,77 @@
 <template>
```

Comment:
We might have some conflicts in this file with my most recent PR as well, though my changes were quite minimal. So make sure yours goes in first :)